cobalt/browser/memory_settings/table_printer_test.cc

```diff
@@ -16,6 +16,7 @@
 
 #include "cobalt/browser/memory_settings/table_printer.h"
 
+#include "base/logging.h"
 #include "starboard/common/log.h"
```

Comment:
Nit: `#include starboard/common/log.h` can be removed here too now, i think.

Reply:
Removed